README.md

```diff
@@ -62,7 +73,7 @@ aedt_test_runner --config-file=config.json --aedt-version=193 --only-reference
 aedt_test_runner --config-file=config.json --aedt-version=222 --reference-file=input/reference_results.json
 ```
 
-### Slurm
+#### Slurm
```

Comment:
I think "Local machine" and "Slurm" should be at the same level with ####, because both are Examples (###) if you don't agree I will accept your change.

Reply:
then move one level here: `#### Generate only reference results`